plugins/com.google.cloud.tools.eclipse.appengine.whitelist/plugin.xml

```diff
@@ -12,4 +13,14 @@
          requiredSourceLevel="1.5">
      </compilationParticipant>
   </extension>
+  
+  <extension 
+      id="blacklistedClass" 
+      name="App Engine Whitelist Marker"
+      point="org.eclipse.core.resources.problemmarkers">
```

Comment:
On Tue, Apr 12, 2016 at 4:24 PM, Brian de Alwis <notifications@github.com>
wrote:

> You could embed the bundle symbolic name directly. By referencing a class
> in that bundle, you insulate the test in the face of bundle renames.


I think we're operating at a different levels here. Which string is this
bundle symbolic name you refer to?


-- 
Elliotte Rusty Harold
elharo@macfaq.com